Java/src/test/java/AirportTest.java

```diff
@@ -74,27 +73,25 @@ public void test3() {
     }
 
     @Test
-    public void testHasAtLeastOneBomberInMilitaryPlanes() {
+    public void testHasAtLeastOneBomberInMilitaryPlane() {
         Airport airport = new Airport(planes);
         List<MilitaryPlane> bomberMilitaryPlanes = airport.getBomberMilitaryPlanes();
         boolean flag = false;
         for (MilitaryPlane militaryPlane : bomberMilitaryPlanes) {
             if ((militaryPlane.getType() == MilitaryType.BOMBER)) {
                 flag = true;
-            }
-            else {
-                Assert.fail("Test failed!");
+                break;
             }
         }
-        // if not failed
+        Assert.assertTrue(flag);
     }
 
     @Test
-    public void testExperimentalPlanesHasClassificationLevelHigherThanUnclassified(){
+    public void testHasAtLeastOneUnclassifiedExperimentalPlane(){
         Airport airport = new Airport(planes);
```

Comment:
We have airport initialization in every test method. Makes sense to use fixture (@Before)